crates/ruff_linter/resources/test/fixtures/ruff/RUF052.py

```diff
@@ -37,6 +37,17 @@ def fun():
     _x = "reassigned global"
     return _x
 
+# (we had a false positive when a global var is used like this in 2 functions)
+_num: int
+
+def set_num():
+    global _num
+    _num = 1
+
+def print_num():
+    print(_num)
```

Comment:
Note: I inserted the test here in the `############### Correct #############` section, seemed right thing to do.

It not being at the end of this file means I got some large diffs in the fixture files.  I reviewed the diffs and there's no (substantive) changes to them, as expected.